src/markdown-pages/docs/galasa-hub.md

```diff
@@ -0,0 +1,37 @@
+---
+path: "/hub"
+title: "Galasa Hub"
+---
+
+Stay up-to-date with all things Galasa with blogs, podcasts and videos from fellow Galasians. 
+
+## Blog: IBM Z Application Continuous Testing using Galasa
+A tutorial focusing on using the Galasa framework for automated application testing on IBM Z applications. You will learn how to setup a Galasa Ecosystem, integrate Eclipse and Jenkins with the Ecosystem and run IBM Z application Galasa test cases in the Ecosystem from an Eclipse and a Jenkins pipeline. <br>
+
+**Author: Anuprakash Moothedath, June 11, 2021**<br>
+*Source: IBM.com*<br><br>
+<a href="https://community.ibm.com/community/user/ibmz-and-linuxone/blogs/anuprakash-moothedath/2021/06/09/ibm-z-application-continuous-testing-using-galasa" target="_blank">
```

Comment:
Instead of IBM.com - can we say IBM Developer?

Reply:
done